docs/libertyDev.md

```diff
@@ -2,6 +2,8 @@
 
 Start a Liberty server in dev mode. This task also invokes the `libertyCreate`, `installFeature`, and `deploy` tasks before starting the server.
 
+N.B. starting in the tech preview in the 3.3.1-SNAPSHOT dev mode also invokes `generate-features`. See [generate-features](generate-features.md) for details.
+
```

Comment:
Consider also adding generateFeatures to the parameters section: https://github.com/OpenLiberty/ci.gradle/blob/main/docs/libertyDev.md?plain=1#L53-L65

Reply:
Before changing the dev documentation I'd like to keep the changes localized as YK suggested and just add the `generateFeatures` parameter to the dev mode part of the generate-features.md.